web/beacon-app/src/features/teams/components/NewUserInvitationPage.tsx

```diff
@@ -31,7 +31,7 @@ export function NewUserInvitationPage({ data }: { data: any }) {
       onSuccess: (_response) => {
         // save invitee_token to session storage
         setCookie('invitee_token', invitee_token as string);
-        navigateTo('/verify-account', { replace: true });
+        navigateTo('/', { replace: true });
```

Comment:
The user should be directed to the verify-account page. If they do not see this, they will not know that they need to check their email to verify their account before they're able to log in.